src/domain/eventSearch/utils.tsx

```diff
@@ -99,6 +107,40 @@ export const getCourseCategoryOptions = (t: TFunction): CategoryOption[] => [
   },
 ];
 
+// todo: fix icons and list of options when defined
+export const getCourseHobbyTypeOptions = (t: TFunction): HobbyTypeOption[] => [
+  {
+    icon: <IconMovies />,
```

Comment:
Hmm are these icons used anywhere? same for the categories? can't remember

Reply:
not defined yet, i will fix it with actual values. even the list is temporary